### DISCUSSION TITLE
Print Screen, Scroll Lock, Pause break keys on the virtual keyboard in Android

### DISCUSSION THREAD

**Discussion by AndrewKusov** [rustdesk/rustdesk#6918] [Feature Request]
Not all of these keys are used every day, but Print Screen is definitely needed very often, mainly to create screenshots. Scroll Lock is mainly used in MS Excel, but sometimes I use it for development as a hotkey that is not occupied by any other hotkey at the moment. So I suggest adding them to the top panel of special keys, there is enough space for them there.

**Comment by Demi256** [comment]
The [menu key](https://en.wikipedia.org/wiki/Menu_key) (also called application key or context menu key) is also missing.
